app/screens/network/Network.tsx

```diff
@@ -323,16 +323,12 @@ const Network = ({ history }) => {
                     text="REDIST INSTALLATION GUIDE."
                   />
                 )}
-                {(isShowMissingLibsMessage ||
-                  nodeError?.type === NodeErrorType.NOT_SPECIFIED) && (
-                  <ErrorMessage compact>
-                    WE HAVE A CHECKLIST THAT CAN HELP YOU DETECT THE ISSUE.{' '}
-                    <Link
-                      style={{ display: 'inline-block' }}
-                      onClick={() => setOpenCheckListModal(true)}
-                      text="OPEN CHECKLIST."
-                    />
-                  </ErrorMessage>
+                {nodeError?.type === NodeErrorType.NOT_SPECIFIED && (
+                  <Link
```

Comment:
<img width="196" alt="Screenshot 2023-11-21 at 00 01 18" src="https://github.com/spacemeshos/smapp/assets/54288612/4d5c1905-c936-4c9d-b622-95d67bfe6488">

based on example we have: 
<img width="344" alt="Screenshot 2023-11-21 at 00 41 56" src="https://github.com/spacemeshos/smapp/assets/54288612/80217231-2791-4e9c-a8ea-bf2eb175d975">

```suggestion
{' '}
                  <Link
```